sos/policies/runtimes/__init__.py

```diff
@@ -199,5 +205,31 @@ def get_logs_command(self, container):
         """
         return "%s logs -t %s" % (self.binary, container)
 
+    def get_copy_command(self, container, path, dest, sizelimit=None):
+        """Generate the command string used to copy a file out of a container
+        by way of the runtime.
+
+        :param container:   The name or ID of the container
+        :type container:    ``str``
+
+        :param path:        The path to copy from the container. Note that at
+                            this time, no supported runtime supports globbing
+        :type path:         ``str``
+
+        :param dest:        The destination on the *host* filesystem to write
+                            the file to
+        :type dest:         ``str``
+
+        :param stdout:      Print the contents of the file to stdout, instead
+                            of writing the file to ``dest``
+        :type stdout:       ``bool``
```

Comment:
Not an argument of the method. `sizelimit` is there instead :)

Reply:
Ah, yep - `stdout` was from an earlier revision. I'll update the docstrings.

Reply:
Updated in latest push.